lib/lotus/action/params.rb

```diff
@@ -91,6 +91,17 @@ def self.whitelisting?
         defined_attributes.any?
       end
 
+      # Overrides the method in Lotus::Validation to build a class that
+      # inherits from Params rather than only Lotus::Validations.
+      #
+      # @since x.x.x
+      # @api private
+      def self.build_validation_class(&block)
+        kls = Class.new(Params)
+        kls.class_eval(&block)
+        kls
+      end
```

Comment:
Instead of overriding this method, perhaps we could add one that is designed to be overridden, like:

```ruby
def nested_superclass
  Params
end
```